### PR TITLE
Feature/iss10 body height2

### PR DESCRIPTION
**Pull request by BirgitSaalfeld**
Dear Pablo, 
please check my mapping of the body height  #65 

Regards,
Birgit

### CODE REVIEW

**Comment by ppazos** [comment]
@BirgitSaalfeld can you please check the conflicts with develop branch?

**Comment by BirgitSaalfeld** [comment]
Dear @ppazos, I reintegrated the current develop branch.

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 1]
```diff
@@ -278,15 +278,17 @@ public void createObservationUsingDefaultProfile() {
...
-            "Default profile is not supported for Observation. One of the following profiles is expected: "
-            + "[http://hl7.org/fhir/StructureDefinition/bodytemp, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/FiO2, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, "
-            + "http://hl7.org/fhir/StructureDefinition/heartrate, "
-            + "https://charite.infectioncontrol.de/fhir/core/StructureDefinition/CoronavirusNachweisTest, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/pregnancy-status, "
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Default profile is not supported for Observation. One of the following profiles is expected: " +
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:281]: @BirgitSaalfeld please fix the indentation to 4 spaces.

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 2]
```diff
@@ -299,15 +301,18 @@ public void createObservationUsingUnsupportedProfile() {
...
-            "Profile http://hl7.org/fhir/StructureDefinition/vitalsigns is not supported for Observation. One of the following profiles is expected: "
-            + "[http://hl7.org/fhir/StructureDefinition/bodytemp, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/FiO2, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, "
-            + "http://hl7.org/fhir/StructureDefinition/heartrate, "
-            + "https://charite.infectioncontrol.de/fhir/core/StructureDefinition/CoronavirusNachweisTest, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/pregnancy-status, "
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Profile http://hl7.org/fhir/StructureDefinition/vitalsigns is not supported for Observation. " +
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:304]: @BirgitSaalfeld please fix the indentation to 4 spaces.

**Review by ppazos — CHANGES_REQUESTED** [review]

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java (hunk 2 above)
Comment (BirgitSaalfeld) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:304]: Four spaces counted from what? I learned programming guidlines like this. Do we have guidlines in this project or is there an auto-formatting we can use?

Comment (ppazos) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:304]: Hi @BirgitSaalfeld please check the 'green' vs. 'red', the red code is 4 spaces to the right from the container line 'Assertions.assertEquals(' while the green is 8 spaces to the right from that.

Comment (ppazos) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:304]: BTW for the format I think we use the SonarLint plugin

- https://www.sonarlint.org/vscode
- https://www.sonarlint.org/intellij

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java (hunk 1 above)
Comment (BirgitSaalfeld) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java]: adapted indentation

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java (hunk 2 above)
Comment (BirgitSaalfeld) [reply src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java]: done

**Comment by BirgitSaalfeld** [comment]
same queston as in the other pull request.

**Review by ppazos — APPROVED** [review]

**Pull request merged**